ajax/libs/labelauty/1.1.0/package.json

```diff
@@ -11,10 +11,10 @@
     "target": "git://github.com/fntneves/jquery-labelauty",
     "fileMap": [
       {
-        "basePath": "source",
+        "basePath": "",
```

Comment:
Sorry but I mean to remove it from filename, and put the files outside the `source` directory, we don't need it, thanks.